interfaces/builtin/fwupd.go

```diff
@@ -213,6 +218,11 @@ const fwupdConnectedPlugSecComp = `
 bind
 `
 
+var bootPartitionDirs = []string{
+	"/boot/efi",
+	"/boot/grub",
```

Comment:
I agree with @zyga: I'd just mount `/boot` here using `rbind` instead of `bind`, which will automatically handle file systems mounted under `/boot` such as `/boot/efi`.

And with only one directory to bind, I'd remove the for loops below and just specify `/boot` directly.